tests/test_git/test_git_init.py

```diff
@@ -0,0 +1,47 @@
+import sublime
+import os
+from .common import GSCreateRepoTestCase
+from GitSavvy.core import git_command
+
+
+def tidy_path(path):
+    return os.path.realpath(os.path.normcase(path))
+
+
+class TestInit(GSCreateRepoTestCase, git_command.GitCommand):
+
+    def test_01_window_path(self):
+        self.assertEqual(
+            tidy_path(self._temp_dir),
+            tidy_path(sublime.active_window().folders()[0]))
+        self.assertEqual(
+            tidy_path(self._temp_dir),
+            tidy_path(self.window.folders()[0]))
+
+    def test_02_add_file(self):
```

Comment:
Each operation "should" be tested separately. For operations that need the app to be at a certain state we could use a fixture of sorts, e.g a collection of directories with git repositories at specific states to be copy-pasted on each related set of tests (e.g in `setUpClass`).

This brings me to another point I would like to consider. I would prefer the tests (and their dependencies) to be separate from the plugin, so that downloading a plugin from package-control will not include any of our internal development files. Now, I'm not sure Package Control supports this degree of separation, so we may need to change our build process to [manual release][1].

[1]: https://packagecontrol.io/docs/creating_package_files